### DISCUSSION TITLE
Implementing Chainlink vrf

### DISCUSSION THREAD

**Discussion by SuryaSundarVadali** [Cyfrin/foundry-full-course-cu#2137] [Q&A] [answered]
In version 1.2.0 in smart contract kit there is no VRFConsumerBaseV2Plus what to do ?

**Comment by EngrPips** [comment]
Hello @SuryaSundarVadali, Have you checked properly? I suggest you use `command + p` for Mac and `cltr + p` for Windows, and a search bar will pop up. Type the name of the file in the search bar, and you will be sure if the `VRFConsumerBaseV2Plus` is in the `1.2.0` or not. If it is not there, then you should install version `0.6.1` using this command `forge install smartcontractkit/chainlink-brownie-contracts@0.6.1 --no-commit`

**Comment by EngrPips** [comment, marked as answer]
<img width="1440" alt="Screenshot 2024-07-30 at 15 00 35" src="https://github.com/user-attachments/assets/279c1ba0-c4a1-4a9d-8bc5-5204a54df6b8">

Also, the latest version is `2.14.0` as shown in the image above, and it has the `VRFConsumerBaseV2Plus` file, so you can instead just run `forge install smartcontractkit/chainlink-brownie-contracts --no-commit` to get the latest.

<img width="1440" alt="Screenshot 2024-07-30 at 15 01 29" src="https://github.com/user-attachments/assets/599f4c04-6eea-4837-8282-18d9c5c40555">

**Reply by SuryaSundarVadali** [reply]
Thanks i have gone back to version 1.1.1 which has VRFConsumerBasePlus2

**Comment by SuryaSundarVadali** [comment]
[
![Github](https://github.com/user-attachments/assets/f645266c-b4cc-4d5d-be59-b979fa4f1637)
![Github](https://github.com/user-attachments/assets/5da770b2-ac35-4302-b31c-c878e6399de3)
](url)

**Reply by EngrPips** [reply]
If you get the file you wanted then we are good! have fun.
